pkg/server/testserver.go

```diff
@@ -1266,6 +1266,29 @@ func (t *testTenant) DeploymentMode() serverutils.DeploymentMode {
 	return t.deploymentMode
 }
 
+// GrantTenantCapability is part of the serverutils.TenantControlInterface.
+func (ts *testServer) GrantTenantCapability(
+	ctx context.Context,
+	tenID roachpb.TenantID,
+	capability tenantcapabilities.ID,
+	capabilityValue string,
```

Comment:
I wonder if we should make this `GrantTenantCapabilities` and take the same map that WaitForTenantCapabilities takes.

Reply:
That helps tip the scale! I’ll update it with this suggestion.